src/Student6/Ex4/Tweet.js

```diff
@@ -0,0 +1,11 @@
+import './Tweet.scss'
+
+export const Tweet = (props) => {
+  return (
+    <div className="Tweet" style={{backgroundColor: props.color}}>
+      {props.message}
+      <button onClick={()=>props.onClick(props.number)}>Like</button> 
```

Comment:
Where is `props.number` defined?  I'm pretty sure we can delete this.

Reply:
I deleted props.number and re ran the code.  Everything is working now